hydrax/simulation/deterministic.py

```diff
@@ -100,6 +106,65 @@ def run_interactive(  # noqa: PLR0912, PLR0915
         pert = mujoco.MjvPerturb()
         catmask = mujoco.mjtCatBit.mjCAT_DYNAMIC  # only show dynamic bodies
 
+    # Initialize video recording if enabled
+    renderer = None
+    process = None
+    if record_video:
+        renderer = mujoco.Renderer(mj_model, height=480, width=720)
+        print("Recording scene...")
+
+        # Set up video writer
+        recordings_dir = os.path.join(ROOT, "recordings")
+        if not os.path.exists(recordings_dir):
+            os.makedirs(recordings_dir)
+
+        timestamp = datetime.now().strftime("%Y%m%d_%H%M%S")
+        video_path = os.path.join(recordings_dir, f"simulation_{timestamp}.mp4")
+
+        # Check if FFmpeg is available
+        try:
```

Comment:
nice feature. i think it might be cleaner to put the video recording logic wrapped under a function located in, e.g., `hydrax/utils/video.py` or something like that.

There should probably also be better default camera settings for some of the example tasks, though that might belong in a cleanup PR.

Reply:
I have moved the video recording logic to a class in `hydrax/utils/video.py`